ZFSin/zfs/module/zfs/zfs_vnops_windows_lib.c

```diff
@@ -3378,7 +3378,7 @@ NTSTATUS ioctl_disk_get_partition_info(PDEVICE_OBJECT DeviceObject, PIRP Irp, PI
 	part->HiddenSectors = (ULONG)(1L);
 	part->RecognizedPartition = TRUE;
 	part->RewritePartition = FALSE;
-	part->PartitionType = 'ZFS';
+	part->PartitionType = 0x7F;	// 'ZFS'; // https://en.wikipedia.org/wiki/Partition_type
```

Comment:
I think this is outside the scope of fixing a compiler warning.  also, ZFS is originally Solaris and some pages on the net are saying it should be 0xBF.  I think you should reserve that change for a different PR so you can first double check with Jorgen the best value here.

Reply:
OK, I will revert it.